packages/core/ios/RNEmbrace/EmbraceManager.swift

```diff
@@ -0,0 +1,301 @@
+import Foundation
+import EmbraceIO
+import EmbraceCore
+import EmbraceCrash
+import EmbraceCommonInternal
```

Comment:
Is necessary to import `EmbraceCommonInternal`? Which is the object/class/method needing this "dependency" to be imported?

Reply:
this might be fixed but it was needed for the `LogSeverity` enum